src/main/java/MAYGEN/MAYGEN.java

```diff
@@ -2803,10 +2823,16 @@ private void parseArgs(String[] args) throws ParseException {
             CommandLine cmd = parser.parse(options, args);
             this.formula = cmd.getOptionValue("formula");
             if (cmd.hasOption("filedir")) {
-                this.writeSDF = true;
+                //this.writeSDF = true;
                 String filedir = cmd.getOptionValue("filedir");
                 this.filedir = Objects.isNull(filedir) ? "." : filedir;
             }
+            if (cmd.hasOption("type")){

```

Comment:
It will be better to support two formats.

It may be keys `-createSdf` and `-createSmiles`